src/check.h.in

```diff
@@ -980,6 +997,7 @@ CK_DLL_EXP void CK_EXPORT srunner_run_all(SRunner * sr,
  */
 CK_DLL_EXP void CK_EXPORT srunner_run(SRunner * sr, const char *sname,
```

Comment:
Making a change to this call will break the API for any user using this call. This call was [introduced in 2010](https://github.com/libcheck/check/commit/f84590bb43dfdf4364765c9c129f12b550440947).

The previous call will run a specific suite or test case. It may make more sense to add a separate call for running all tests for a given tag. Under the covers the new call and srunner_run() may call some other method which takes all the arguments for test case names, tags, etc.